docs/developer/payments/overview.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Payments Overview
+sidebar_label: Overview
+---
+
+There are few ways to work with payments in Saleor:
+
+### Payment apps
+
+Payment apps allow to process payments with external payment providers, without communicating with them directly, which is specially convenient for front-end development.
+
+- Using [payment app](/developer/payments/payment-apps.mdx).
+
+- Guide to [building payment apps](/developer/extending/apps/building-payment-app.mdx).
+
+:::note
+
+If you're considering building a custom payment integration, please check if the payment provider you're interested in isn't already available [Saleor App Store](developer/app-store/apps/overview.mdx#built-in-apps-catalog).
+
+:::
+
+### Using plugins included in core
+
+Using payment plugins is similar to using payment apps, with few key differences:
+
+- Payment plugins are included in core, so it is discouraged to modify them.
+- Building your own payment plugins is discouraged, and relies on undocumented APIs which will be deprecated.
+- Payment plugins are not as flexible as payment apps, and do not support multiple transactions on a single order.
+
+**Should you use payment plugins?**
+
+In case you need to use a payment provider that is not [available as a payment app](developer/app-store/apps/overview.mdx#built-in-apps-catalog), you can use a payment plugin. In case such provider will be available in the future swapping plugin with app would require refactoring of the checkout flow.
+
+### Payment APIs via Backend
```

Comment:
Based on the context it is like "more advanced use case". Maybe worth considering a different name for this section? I totally didn't know what it was at the beginning - something like "advanced use case"?.

Also as far as I understand, from the description this covers more advanced use cases for handling a payments. I believe in that case, we should revert the `transactionCreate` from deprecated section: https://github.com/saleor/saleor-docs/pull/1248/files#r1688280173 as `transactionCreate`/`transcationUpdate` are the mutations which gives you a flexibility to cover such cases.

Reply:
@korycins could you please take a look at a new overview page, perhaps it is less confusing now